src/elm/Community.elm

```diff
@@ -116,6 +118,8 @@ type alias Model =
     , hasKyc : Bool
     , hasAutoInvite : Bool
     , validators : List Eos.Name
+    , coverPhoto : Maybe String
```

Comment:
This should be a list

Reply:
Yep, I fixed it today, just haven't pushed it yet